code/modules/projectiles/guns/mounted.dm

```diff
@@ -457,6 +457,13 @@
 	. = ..()
 	sponson = new sponson(src)
 
+/obj/machinery/deployable/mounted/moveable/atgun/attackby(obj/item/I, mob/user, params)
+	if(user.interactee)
```

Comment:
this is just checking if the user is interacting with anything, not src specifically.

Reply:
Is this okay?